redash/query_runner/jql.py

```diff
@@ -179,6 +181,19 @@ def run_query(self, query, user):
                 results = parse_count(data)
             else:
                 results = parse_issues(data, field_mapping)
+                index = data['startAt'] + data['maxResults']
+
+                while data['total'] > index:
+                    query['startAt'] = index
+                    response, error = self.get_response(jql_url, params=query)
+                    if error is not None:
+                        return None, error
+
+                    data = response.json()
+                    index = data['startAt'] + data['maxResults']
+
+                    addl_results = parse_issues(data, field_mapping)
```

Comment:
Is it possible that the next pages of results might have different columns than the previous ones?

Reply:
Looking at [their docs](https://developer.atlassian.com/server/jira/platform/rest-apis/#pagination) it appears `startAt`, `maxResults`, and `total` will always be in the response.

Reply:
Ah, I see. That wouldn't be possible. They are additional results to the same query. The query defines the columns. If the column is empty it'll still come back but with a null value.